tests/v2/validation/rbac/rbac_test.go

```diff
@@ -162,28 +204,71 @@ func (rb *RBTestSuite) ValidateNS(role string) {
 	}
 }
 
+func (rb *RBTestSuite) ValidateAddClusterRoles(role string) {
+
+	//Testcase7 Validate if project members are able to add other membes in cluster
+	errUserRole := users.AddClusterRoleToUser(rb.standardUserClient, rb.cluster, rb.additionalUser, role)
+
+	switch role {
+	case roleProjectOwner, roleProjectMember:
+		require.Error(rb.T(), errUserRole)
+		assert.Equal(rb.T(), true, k8sErrors.IsForbidden(errUserRole))
+	}
+}
+
+func (rb *RBTestSuite) ValidateAddProjectRoles(role string) {
+
+	//Testcase8 Validate if project owners/members are able to add another standard user as a project members
+	errUserRole := users.AddProjectMember(rb.standardUserClient, rb.adminProject, rb.additionalUser, role)
+
+	switch role {
+	case roleProjectOwner:
+		require.NoError(rb.T(), errUserRole)
+		additionalUserClient, err := rb.additionalUserClient.ReLogin()
+		require.NoError(rb.T(), err)
+		rb.additionalUserClient = additionalUserClient
+
+		projectList, err := listProjects(rb.standardUserClient, rb.cluster.ID)
+		require.NoError(rb.T(), err)
+		assert.Equal(rb.T(), 1, len(projectList))
+		assert.Equal(rb.T(), rb.adminProject.Name, projectList[0])
+	case roleProjectMember:
+		require.Error(rb.T(), errUserRole)
+	}
+
+}
+
 func (rb *RBTestSuite) ValidateDeleteProject(role string) {
 
-	//Testcase7 Validate if cluster members are able to delete the project they are not owner of
+	//Testcase9 Validate if cluster members are able to delete the admin created project
 	err := rb.standardUserClient.Management.Project.Delete(rb.adminProject)
 
 	switch role {
-	case roleOwner:
+	case roleOwner, roleProjectOwner:
 		require.NoError(rb.T(), err)
 	case roleMember:
 		require.Error(rb.T(), err)
 		errStatus := strings.Split(err.Error(), ".")[1]
 		rgx := regexp.MustCompile(`\[(.*?)\]`)
 		errorMsg := rgx.FindStringSubmatch(errStatus)
 		assert.Equal(rb.T(), "403 Forbidden", errorMsg[1])
+	case roleProjectMember:
+		require.Error(rb.T(), err)
 	}
 }
 
-func (rb *RBTestSuite) ValidateRemoveClusterRoles(role string) {
+func (rb *RBTestSuite) ValidateRemoveClusterRoles() {
 
-	//Testcase8 Remove added cluster member from the cluster as an admin
+	//Testcase10a Remove added cluster member from the cluster as an admin
 	err := users.RemoveClusterRoleFromUser(rb.client, rb.standardUser)
 	require.NoError(rb.T(), err)
+}
+
+func (rb *RBTestSuite) ValidateRemoveProjectRoles() {
+
+	//Testcase10b Remove added cluster member from the cluster as an admin
```

Comment:
just a nit, I think this comment is wrong / copied from the previous function

Reply:
Updated